perfkitbenchmarker/alicloud/ali_network.py

```diff
@@ -85,41 +210,13 @@ def _Exists(self):
     stdout, _ = vm_util.IssueRetryableCommand(show_cmd)
     return 'SecurityGroupId' in json.loads(stdout)
 
-  def AllowPort(self, port):
-    for proto in ('tcp', 'udp'):
-      allow_cmd = util.ALI_PREFIX + [
-          'ecs',
-          'AuthorizeSecurityGroup',
-          '--IpProtocol %s' % proto,
-          '--PortRange %s/%s' % (port, port),
-          '--SourceCidrIp 0.0.0.0/0',
-          '--RegionId %s' % self.region,
-          '--SecurityGroupId %s' % self.group_id]
-      allow_cmd = util.GetEncodedCmd(allow_cmd)
-      vm_util.IssueRetryableCommand(allow_cmd)
 
 
 class AliFirewall(network.BaseFirewall):
```

Comment:
Given the fact that you are now opening up all ports when you create the VM, I don't think you need this anymore